javascript-carousel/main.js

```diff
@@ -0,0 +1,80 @@
+const $imageRow = document.querySelector('#image-row');
+const $circleRow = document.querySelector('#circle-list');
+const $circleList = $circleRow.children;
+const $pokemonImage = document.querySelector('#pokemon-img');
+
+var intervalID = setInterval(scrollRight, 3000);
+
+$imageRow.addEventListener('click', function (event) {
+  if (event.target.id !== 'left' && event.target.id !== 'right') return;
```

Comment:
To avoid this check, you could just directly add listeners to the `id="left"` and `id="right"` elements.

Reply:
Ok, added listeners to the two left and right and simplified the listener